aten/src/ATen/native/quantized/cpu/fbgemm_utils.h

```diff
@@ -198,6 +210,10 @@ struct CAFFE2_API PackedConvWeight : public ConvPackedParamsBase<kSpatialDim> {
     return groups_;
   }
 
+  bool transpose() const override {
+    return (bool)transpose_;
```

Comment:
why do we need type conversion here?

Reply:
`transpose_` is `uint8_t` (doesn't work if I use `bool`). Just using explicit conversion